src/main/resources/META-INF/spring-configuration-metadata.json

```diff
@@ -3,6 +3,11 @@
     {
       "name": "lsd.dist.obfuscator.sensitiveHeaders",
       "type": "java.lang.String"
+    },
+    {
+      "name": "lsd.dist.queue.length",
```

Comment:
just wondering about the name of the property 🤔 , queue length threw me a bit - I had to double check that it was actually the pool size and not a limit on number of events or something like that.

Reply:
I agree. Any suggestions?